modules/ROOT/pages/clustering/setup/deploy.adoc

```diff
@@ -253,6 +253,13 @@ If you want to follow along with the startup, you can see the messages in xref:c
 ======
 =====
 
+[CAUTION]
+====
+The setting
+xref:configuration/configuration-settings.adoc#dbms.cluster.minimum_initial_system_primaries_count[`dbms.cluster.minimum_initial_system_primaries_count`]
+must be set to `2` on all servers in case setting up a cluster with only _two_ servers.
```

Comment:
```suggestion
must be set to `2` on all servers in case setting up a cluster with only *two* servers.
```

Reply:
We try to use bold if you need to highlight anything. Italics is for file names.